src/helix-ui/styles/elements/hx-popover.less

```diff
@@ -38,6 +38,7 @@ hx-popover {
   }
 
   & > &-foot {
+    display: flex;
```

Comment:
Please move to `src/helix-ui/styles/elements/hx-popover-foot.less`. This style should always apply, regardless of if it's within a `<hx-popover>`. 

Also, this location results in a CSS selector with higher specificity (making it more difficult to override) than if it were placed in the other file (`hx-popover > hx-popover-foot` vs `hx-popover-foot`). If we can achieve the same styles with less complexity, we should opt for the less complex solution.

Reply:
> If we can achieve the same styles with less complexity, we should opt for the less complex solution.

I fully agree 💯